content/rooms/v11.md

```diff
@@ -241,7 +243,7 @@ Some consequences of these rules:
 
 ### Redactions
 
-{{% rver-fragment name="v11-redactions" withVersioning="true" %}}
+[See above](#redactions).
```

Comment:
(it's specifically supposed to be duplicated, not copied - this is where use of fragments comes into play)

Reply:
I copied what room version 9 does: https://spec.matrix.org/v1.7/rooms/v9/#redactions-1 I guess I'll fix that too?

Reply:
Actually, all the room versions do this:

https://spec.matrix.org/v1.7/rooms/v1/#redactions-1

https://spec.matrix.org/v1.7/rooms/v6/#redactions-1

Reply:
Let me know if you think I should update them all or leave them all. 👍

Reply:
oh, let's just do what v9 and the others do.